src/dotnet/commands/dotnet-restore/NuGet3.cs

```diff
@@ -10,13 +10,13 @@ namespace Microsoft.DotNet.Tools.Restore
 {
     internal static class NuGet3
     {
-        public static int Restore(IEnumerable<string> args, bool quiet)
+        public static int Restore(IEnumerable<string> args)
         {
             var prefixArgs = new List<string>();
-            if (quiet)
+            if (!args.Any(s => s.Equals("--verbosity", StringComparison.OrdinalIgnoreCase) || s.Equals("-v", StringComparison.OrdinalIgnoreCase)))
             {
                 prefixArgs.Add("--verbosity");
-                prefixArgs.Add("Error");
+                prefixArgs.Add("minimal");
```

Comment:
I agree "Information" is too spammy.

I would really like to see NuGet "do the right thing here", and not have this "default" in CLI.

@yishaigalatzer @emgarten @rrelyea  - care to comment?